lib/helpers/get-login-view-model.js

```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+var _ = require('lodash');
+
+var cachedViewModel;
+var cacheTimestamp = 0;
+
+/**
+ * Returns the cached model if it's not older than `ttl`.
+ * Else it returns null.
+ *
+ * @method
+ *
+ * @param {number} ttl - The time-to-live value for the cached object.
+ */
+function getCachedModel(ttl) {
+  var age = (new Date().getTime() - cacheTimestamp) / 1000; // In seconds.
```

Comment:
The `ttl` is in seconds, so the conversion must happen here or in the `if` two lines down. I prefer to normalize the age here so we have both `ttl` and `age` in seconds. Or am I missing something?